functional_test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+if [ "$#" -lt 1 ]; then
+    echo "Usage: functional_test.sh TAG"
+    exit 1
+fi
+
+TAG=$1
+
+TMPDIR=$(mktemp -d)
+
+# Import cmd
+docker run \
+    -v "$(pwd)/web/test-data/:/graphhopper/test-data/" \
+    -v "$TMPDIR:/graphhopper/transit_data/"\
+    --rm \
+     us.gcr.io/model-159019/gh:"$TAG" \
+     java -Xmx2g -Xms1g -XX:+UseG1GC -XX:MetaspaceSize=100M \
+     -Ddw.graphhopper.datareader.file=test-data/kansas-city-extract-mini.osm.pbf \
+     -Ddw.graphhopper.gtfs.file=test-data/mini_kc_gtfs.tar \
+     -classpath web/target/graphhopper-web-1.0-SNAPSHOT.jar -server com.graphhopper.http.GraphHopperApplication import default_gh_config.yaml
+
+# Run server in background
+docker run --rm  --name functional_test_server -p 50051:50051 -p 8998:8998 -v "$TMPDIR:/graphhopper/transit_data/" \
+    us.gcr.io/model-159019/gh:"$TAG" &
+
+echo "Waiting for graphhopper server to start up"
+sleep 30
+
+# Make a request for a couple of points in the minikc region
+grpcurl -d @ -plaintext localhost:50051 router.Router/RouteStreetMode <<EOM
```

Comment:
It would also be great if we could test a transit request here (those are the ones we have issues with more often). As you've pointed out, mini_kc is a tiny region, so I think using the exact transit requests I added to the unit tests recently would be best here (it's one of the only transit requests I could get to return a result, given how small the region is 😂 ).

This will also mean you'll need to run the GTFS link mapper step after the current `import` step runs, because transit responses require link mappings to be present so we can return network link IDs for bus/tram routes. All you should need to add is something like 
```
<current java command> && java -Xmx2g -Xms1g -XX:+UseG1GC -XX:MetaspaceSize=100M -classpath web/target/graphhopper-web-1.0-SNAPSHOT.jar com.graphhopper.http.GraphHopperApplication gtfs_links default_gh_config.yaml
```